src/main/scala/com/codacy/clangtidy/Main.scala

```diff
@@ -0,0 +1,15 @@
+package com.codacy.clangtidy
+
+import io.circe.generic.AutoDerivation
```

Comment:
Why circe is spilling to the `Main`?
Also we should avoid completely automatic derivation as it has numerous drawbacks

Reply:
It was a leftover from playing around in the main class. Forgot to remove it in the refactor

Reply:
Fixed